test/testcafe/regression-tests.js

```diff
@@ -35,7 +35,7 @@ const setProdPerfectCookie = ClientFunction( (id, name, testSuiteRunId, env) =>
 
 
 fixture `Test Page`
-  .page `https://cbracco.github.io/html5-test-page/`
+  .page `./html5-test-page.html`
```

Comment:
I can't think of a reason it would be a problem.

It is important that `RequestMock` governs behaviour in the test-cafe proxy layer and does not stub actual browser APIs. We don't want to be in a situation where testcafe implements this via a stub of `XMLHttpRequest` or the fetch api because: a) form submission is likely not to use these, and b) we realllllly want the actual browser behavior, a stub might cause a test to pass where the actual browser, unstubbed might fail.

I suppose the reason to use `RequestMock` is if setting up a server or the like is difficult. If it is possible / easy / clean to just start and stop a plain-old http server in the tests, let's do that.